java/test/jmri/implementation/DefaultMemoryTest.java

```diff
@@ -26,6 +26,15 @@ public void testSystemNames() {
         Assert.assertEquals("Memory system name is correct", "IM01", myMemory_2.getSystemName());
         Assert.assertNotEquals("Memory's are different", myMemory_1, myMemory_2);
         Assert.assertNotEquals("Memory compareTo returns not zero", 0, myMemory_1.compareTo(myMemory_2));
+        
+        Assert.assertEquals("compareSystemNameSuffix returns correct value",
```

Comment:
Since this is testing a method of the class, please make it a separate test, rather than lumping it in with another test.